src/client/src/Domains/Student/StudentProfileForm/index.tsx

```diff
@@ -133,6 +140,28 @@ function StudentProfileForm() {
                                         component={TextFormField}
                                     />
                                 </Grid>
+                                <Grid item md={6} xs={12}>
+                                    <Field
+                                        name='resume'
+                                        label='Resume'
+                                        type='file'
+                                        InputLabelProps={{
```

Comment:
Do we need the InputLbaelProps here? just making sure.

Reply:
I believe we do because if it's not there, we can't see the resume and transcript labels.